packages/test-helpers/src/setup.js

```diff
@@ -41,6 +44,7 @@ const webHooks = () => {
         "App did not start. Run 'fructose-web --build-dir path/to/dir' first"
       );
     }
+    global.fructoseClient = fructoseClient(7811);
```

Comment:
can we return the client as the resolution of the promise rather than setting it as a global.